external-scripts.json

```diff
@@ -2,7 +2,6 @@
   "hubot-diagnostics",
   "hubot-help",
   "hubot-heroku-keepalive",
-  "hubot-google-images",
```

Comment:
Oh my mistake. I think i was really tired looking at it. I thought woz image didnt work anymore